dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/utils/MessageConstants.java

```diff
@@ -12,17 +12,69 @@
 
 public class MessageConstants {
 
+    /**
+     * CWWKO1652W: The server cannot read the specified CSS document {0} due to {1} : {2}.
```

Comment:
The CWWKO prefix is already taken;

CWWKO | orb | WebSphere Application Server - Alpine project
https://github.ibm.com/was-liberty/WS-CD-Open/wiki/CWW-Message-Prefixes
(IBM internal url)

Reply:
I think we need a separate issue for this as it's also used throughout the mpOpenAPI-1.0 and 1.1 features. https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/resources/OpenAPI.nlsprops#L18

Reply:
https://github.com/OpenLiberty/open-liberty/issues/15108